.swiftlint.yml

```diff
@@ -2,7 +2,36 @@ disabled_rules:
  - type_name
 
 opt_in_rules:
+ - array_init
```

Comment:
I didn't know that rule, seems to be for preventing to do `seq.map { $* }` instead of `Array(seq)`, I didn't know some people would do the former 😄 :+1: